MilanAmbrosiana/VBAx105sup.xml

```diff
@@ -165,14 +165,15 @@
         </profileDesc>
 <revisionDesc>
 
-<change who="ES" when="2023-05-17">Created stub</change>
+    <change who="ES" when="2023-05-17">Created stub</change>
+    <change who="MV" when="2024-12-20">Added IDs</change>
```

Comment:
as it is not properly a stub any more you can modify the previous change element (`<change who="ES" when="2023-05-17">Created basic initial version</change>`)